deploifai/workspace/list.py

```diff
@@ -0,0 +1,32 @@
+import click
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated,
+)
+from deploifai.utilities.user import parse_user_profiles
+
+
+@click.command()
+@pass_deploifai_context_obj
+@is_authenticated
+def list(context: DeploifaiContextObj):
+    """
+    Lists out all available workspaces
+    """
```

Comment:
list as a function name might cause errors in python
please suggest an alternate name